index.bs

```diff
@@ -28,6 +28,13 @@ Status Text: <p class="note" role="note">
   further improvements are expected to be reflected in revised Candidate
   Recommendation Drafts and Snaphots.
   </p>
+  <p>Before requesting transition to <a href="https://www.w3.org/standards/types#PR">Proposed Recommendation</a>, the Working Group will seek to demonstrate that:</p>
+  <ul>
+  <li>the API is  implementable on top of existing major platform APIs, such as Android Neural Networks API, Windows DirectML, and macOS/iOS Metal Performance Shaders and Basic Neural Network Subroutines;</li>
```

Comment:
@dontcallmedom

Reply:
given the discussion in https://github.com/webmachinelearning/webnn/pull/362/files#r1145693621, should "macOS/iOS Metal Performance Shaders and Basic Neural Network Subroutines"  be replaced with ML Compute?

I agree with you that the list can only be reasonably read as informative; but if we already know better, we should use the best of our knowledge.

Reply:
To my understanding the more recent ML Compute uses MPS and BNNS so it could be an alternative implementation path on Apple platforms. I’d prefer wording that allows Apple’s ML experts to use any of these platform APIs for their possible implementation and let that be considered another implementation in this context.